test/evergreen.yml

```diff
@@ -4002,9 +4001,6 @@ buildvariants:
   run_on:
   - ubuntu2004-wt-build
   expansions:
-    configure_env_vars: PATH=/opt/mongodbtoolchain/v3/bin:$PATH
```

Comment:
The v3 toolchain contains a clang-7 binary which isn't present in the v4 toolchain. 
Our older versions of clang (8.0, 7.0) want to use the platform toolchains and not the mongo toolchain, so don't export these paths unless we're building for the mongotoolchain.